chart/localdb/charts/dbdump/templates/dbdump-job.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: compass-dbdump
```

Comment:
Can we add the namespace here explicitly as well? The file seems more self contained.